tests/robot-tests/tests/general_public/table_tool_exclusions_by_geographic_level.robot

```diff
@@ -33,8 +33,71 @@ Select subject "Exclusions by geographic level"
+Select all LA Locations
     user opens details dropdown    Local Authority
+    user clicks button    Select all 156 options
+    user clicks element    id:locationFiltersForm-submit
+    user waits until table tool wizard step is available    4    Choose time period
+    user checks previous table tool step contains    3    Local Authority    Barking and Dagenham
+    user checks previous table tool step contains    3    Local Authority    Barnet
+    user checks previous table tool step contains    3    Local Authority    Barnsley
+    user checks previous table tool step contains    3    Local Authority    Bath and North East Somerset
+    user checks previous table tool step contains    3    Local Authority    Bedford
+    user checks previous table tool step contains    3    Local Authority    Show 151 more items
+
+Select all available Time periods
+    user chooses select option    id:timePeriodForm-start    2006/07
+    user chooses select option    id:timePeriodForm-end    2016/17
+    user clicks element    id:timePeriodForm-submit
+    user waits until table tool wizard step is available    5    Choose your filters
+    user waits until page contains element    id:filtersForm-indicators
+    user checks previous table tool step contains    4    Time period    2006/07 to 2016/17
+
+Select Indicator - Number of pupils
+    user clicks indicator checkbox    Number of pupils
+    user checks indicator checkbox is checked    Number of pupils
+
+Select Indicator - Number of permanent exclusions
+    user clicks indicator checkbox    Number of permanent exclusions
+    user checks indicator checkbox is checked    Number of permanent exclusions
+
+Select Indicator - Number of fixed period exclusions
+    user clicks indicator checkbox    Number of fixed period exclusions
+    user checks indicator checkbox is checked    Number of fixed period exclusions
+
+Select Indicator - Number of schools
+    user clicks indicator checkbox    Number of schools
+    user checks indicator checkbox is checked    Number of schools
+
+Select Indicator - Permanent exclusion rate
+    user clicks indicator checkbox    Permanent exclusion rate
+    user checks indicator checkbox is checked    Permanent exclusion rate
+
+Select Characteristic School types
+    user opens details dropdown    School type
+    user clicks category checkbox    School type    State-funded primary
+    user clicks category checkbox    School type    State-funded secondary
+    user clicks category checkbox    School type    Special
+
+User clicks Create table button
+    user clicks element    id:filtersForm-submit
+
+Validate the query could exceed the maximum allowable table size
+    user waits until element contains    id:filtersForm-indicators-error
+    ...    A table cannot be returned as the filters chosen can exceed the maximum allowable table size
+
+Go back to Locations step
+    user clicks element    xpath://button[contains(text(), "Edit locations")]
```

Comment:
We could use `user clicks button  Edit locations` to be a bit more succinct here

Reply:
Good suggestion, I've made this change now, thanks